content/en/continuous_integration/troubleshooting.md

```diff
@@ -183,6 +183,20 @@ The pipeline page only displays pipelines with no Git information, or pipelines
 
 Missing stages or jobs in the _Pipeline Details_ page might be due to a wrong configuration. Make sure that the pipeline name stored in the stage or job executions matches the **same** name of their parent pipeline. If you are using custom pipelines, refer to the [public API endpoint specification][15].
 
+### Limitations on Running Pipelines
+
+#### Delivery of webhook events is not ensured by CI providers
+
+Running pipelines support relies on data sent from CI providers indicating execution status. If this data is not available, executions marked as `Running` in Datadog may have already finished. 
+
+#### Maximum duration for a pipeline execution
+
+The maximum duration that a pipeline execution can maintain `Running` status is 3 days. If a pipeline execution finishes after 3 days, you can visualize the finished pipeline execution in CI Visibility.
```

Comment:
With what status? @drodriguezhdez 

(made some edits)

Reply:
```suggestion
The maximum duration that a pipeline execution can maintain `Running` status is 3 days. If a pipeline execution finishes after 3 days, you can visualize the finished pipeline execution as usual in CI Visibility with its correspondent final status (`Success`, `Error`, `Canceled`, `Skipped`, etc).
```

If a pipeline execution finishes after 3 days, you will see the pipeline finished with the correspondent status (success, failed, canceled, skipped, etc).  There's no special status for this case.